build.py

```diff
@@ -33,7 +33,9 @@
     {'name': 'nodejs-dev', 'tag': 'bionic-dubnium',
         'path': 'nodejs-dev/dubnium', 'args': {'BASE_TAG': 'bionic'}},
     {'name': 'c-dev', 'tag': 'bionic'},
-    {'name': 'devops', 'tag': 'bionic'}
+    {'name': 'devops', 'tag': 'bionic'},
+    {'name': 'deno-dev', 'tag': 'v1.0', 'path': 'deno-dev/v1.0',
```

Comment:
Please follow the same convention as the ruby and python tags.

Reply:
should I remove the paths then and just use `deno-dev`